core/ceremony/ceremony.go

```diff
@@ -205,12 +210,35 @@ func (vc *ValidationCeremony) LongSessionFlipsCount() uint {
 
 func (vc *ValidationCeremony) restoreState() {
 	vc.generateFlipKeyWordPairs(vc.appState.State.FlipWordsSeed().Bytes())
-	timestamp := vc.epochDb.ReadShortSessionTime()
-	if timestamp != nil {
-		vc.appState.EvidenceMap.SetShortSessionTime(timestamp, vc.config.Validation.GetShortSessionDuration())
+
+	if vc.appState.State.ValidationPeriod() >= state.FlipLotteryPeriod {
```

Comment:
why `if` here? parameters are static, `SetShortSessionTime` does not run any timers